accounts/models.py

```diff
@@ -1,10 +1,33 @@
 from django.db import models
-from django.contrib.auth.models import User
+from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.admin import UserAdmin
+from django.contrib import admin
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
 from edumanage.models import Institution
+import re
 
 
+class User(AbstractUser):
+    class Meta(AbstractUser.Meta):
+        swappable = 'AUTH_USER_MODEL'
+        db_table = 'auth_user';
+        verbose_name = _('user')
+        verbose_name_plural = _('users')
+
+User._meta.get_field('username').max_length = 255
+# try also updating the help_text
+try:
+    help_text = User._meta.get_field('username').help_text._proxy____args[0]
+    if help_text != None:
+	User._meta.get_field('username').help_text = _(re.sub("[0-9]+ characters", "255 characters", help_text))
+except:
+    pass
+
+admin.site.register(User, UserAdmin)
```

Comment:
Hi Zenon,

Good point!  I've checked and the Django auth.User model also does the registration from admin.py.

Done, pushed in in 3e00840.

We should be good to go now!

Cheers,
Vlad